client/lib/abtest/active-tests.js

```diff
@@ -94,5 +94,13 @@ module.exports = {
 			targetedWording: 50
 		},
 		defaultVariation: 'originalWording'
-	}
+	},
+	paidNuxStreamlined: {
+		datestamp: '20160825',
+		variations: {
+			original: 100,
+			streamlined: 0,
```

Comment:
Yup, want to leave it inactive for now, since we want to add two additional PRs to the test before we run it. Thanks for checking back!